packages/core/src/components/axes/tool-bar.ts

```diff
@@ -0,0 +1,325 @@
+// Internal Imports
+import { Component } from "../component";
+import { Events } from "../../interfaces";
+import { Tools } from "../../tools";
+import { DOMUtils } from "../../services";
+import { ChartModel } from "../../model";
+import * as Configuration from "../../configuration";
+
+// D3 Imports
+import { event, select } from "d3-selection";
+
+// import the settings for the css prefix
+import settings from "carbon-components/es/globals/js/settings";
+
+export class ToolBar extends Component {
+	type = "tool-bar";
+
+	isZoomBarEnabled: boolean;
+
+	// ul options list element
+	overflowMenuOptions;
+
+	// overflow menu icon rect to control background
+	overflowMenuIcon: any;
+
+	resetZoomMenuItemId = "resetZoomMenuItem";
```

Comment:
what if there are multiple charts on 1 page?

Reply:
Good catch!
Update in https://github.com/carbon-design-system/carbon-charts/pull/717/commits/ba21aeb9d6dbecec4555dd58afd12bc9721fec72.